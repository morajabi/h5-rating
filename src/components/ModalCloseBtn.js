import React, { Component } from 'react'
import styled from 'styled-components'

import { Cross } from '../components/Svgs'

class ModalCloseBtn extends Component {
  render() {
    return (
      <Button>
        <Cross />
      </Button>
    )
  }
}

export default ModalCloseBtn

const Button = styled.button`
  padding: 7px;
  vertical-align: middle;
  line-height: 1;

  opacity: 0.8;
  border-radius: 5px;
  cursor: pointer;
  /* reset button styles */
  background: none;
  border: none;

  transition: background 150ms, opacity 150ms;

  &:hover,
  &:focus {
    background: rgba(0, 0, 0, 0.07);
    opacity: 1;
  }

  &:focus {
    outline: none;
    box-shadow: 0 0 0 2px rgba(0, 0, 0, 0.3);
  }
`
