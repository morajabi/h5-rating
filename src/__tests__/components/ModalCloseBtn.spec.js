import React from 'react'
import { shallow } from 'enzyme'
import ModalCloseBtn from '../../components/ModalCloseBtn'

it('ModalCloseBtn renders as expected', () => {
  const wrapper = shallow(<ModalCloseBtn />)
  expect(wrapper).toMatchSnapshot()
})
