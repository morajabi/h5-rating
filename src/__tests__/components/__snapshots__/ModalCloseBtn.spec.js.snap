// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalCloseBtn renders as expected 1`] = `
<styled.button>
  <Cross />
</styled.button>
`;
